Sources/TuistCache/Utilities/XCFrameworkBuilder.swift

```diff
@@ -71,7 +71,7 @@ public final class XCFrameworkBuilder: XCFrameworkBuilding {
 
     // swiftlint:disable:next function_body_length
     fileprivate func build(_ projectTarget: XcodeBuildTarget, target: Target) throws -> Observable<AbsolutePath> {
-        if target.product != .framework {
+        guard [.framework, .staticFramework].contains(target.product) else {
```

Comment:
nit: we could have a `isFramework` on  `product` to avoid this double checks in many places

Reply:
Good point! I added this isFramework function to Product 👍